[8.0] geoengine_geoname_geocoder: new parent for wizard menu, buttons in footer

alexalv opened the pull request:
Fixes issue #36 
Changed 'parent' for menuitem which is calling partner selection wizard in geoengine_geoname_geocoder module. The same wizard is now called with target set to 'new', boolean 'encode_all' field is now in 'group' tags on wizard form, wizard buttons are now inside 'footer' tag.
![Result](https://cloud.githubusercontent.com/assets/450994/14826358/2d671946-0be6-11e6-8a98-617ad0d53c82.png)

Commit:
Localize addresses from GeoNames wizard menu parent change, target 'new' added in corresponding action

alexalv committed:
geoengine_geoname_geocoder: group for boolean field and buttons in footer for wizard

yvaucher commented:
Thanks for the fix. Could you also:
- add `class="oe_hightlight"` on the button "Encode addresses"
- add `class="oe_link"` on the cancel button
- place cancel on the right as in all other wizards
- add an "or" between the 2 buttons

e.g.: https://github.com/OCA/OCB/blob/8.0/addons/stock/wizard/make_procurement_view.xml#L29-L31

Otherwise tested on runbot the wizard opens correctly.

alexalv commented:
@yvaucher thank you for the fast reply. Done.
![Result](https://cloud.githubusercontent.com/assets/450994/14827891/3281c41a-0bed-11e6-89c5-8fe208d5b0e1.png)

alexalv committed:
geoengine_geoname_geocoder: buttons appearance in wizard corrected.
        Button places in footer switched, 'or' added in between.
        Correct appearance classes added, icons removed.

yvaucher commented on geoengine_geoname_geocoder/wizard/bulk_encode_view.xml:
```diff
@@ -9,21 +9,24 @@
-                        <button icon="gtk-ok" name="encode" string="Encode addresses" type="object"/>
-                    </group>
+                    <footer>
+                        <button class="oe_hightlight" name="encode" string="Encode addresses" type="object"/>
```
Sorry there was a typo in my previous comment. It should be `oe_highlight` there is an extra `t`

alexalv commented:
@yvaucher I completely missed it too. Now fixed. 
![Result](https://cloud.githubusercontent.com/assets/450994/14829171/d518e596-0bf2-11e6-9107-045151d0e50b.png)

alexalv committed:
geoengine_geoname_geocoder: oe_highlight class fixed

yvaucher commented:
:+1: Thanks

yvaucher commented:
Okay let's merge this it seems that we won't get a second review here.
